src/Database/Expression/AggregateExpression.php

```diff
@@ -179,9 +184,9 @@ public function sql(ValueBinder $generator): string
         $sql = parent::sql($generator);
         if ($this->window !== null) {
             if ($this->window->isNamedOnly()) {
-                $sql .= ' OVER ' . $this->window->sql($generator);
+                $sql .= " {$this->clause} " . $this->window->sql($generator);
```

Comment:
What is the point of this change?

Reply:
MS SQL Server uses a syntax similar to SQLite for the `ORDER BY` clause for result sorting within certain aggregate functions but instead of using the `OVER` clause, it uses `WITHIN GROUP`. I modified `AggregateExpression` to make the `OVER` clause dynamic so that it could be changed for TSQL.

SQL Server uses `STRING_AGG(field1, ',') WITHIN GROUP (ORDER BY field2 DESC)` rather than
`GROUP_CONCAT(field1, ';') OVER (ORDER BY field2 DESC)` for SQLite.

https://docs.microsoft.com/en-us/sql/t-sql/functions/string-agg-transact-sql?view=sql-server-ver15

Reply:
This clause is specifically for window expressions.